pennylane/ops/qubit/observables.py

```diff
@@ -314,26 +313,26 @@ def compute_matrix(basis_state):  # pylint: disable=arguments-differ
         m[idx, idx] = 1
         return m
 
-    @classmethod
-    def _eigvals(cls, *params):
-        """Eigenvalues of the specific projector operator.
+    @staticmethod
+    def compute_eigvals(basis_state):  # pylint: disable=,arguments-differ
+        """Eigenvalues of the Projector operator.
+
+        Args:
+            basis_state (Iterable): basis state to project on
 
         Returns:
-            array: eigenvalues of the projector observable in the computational basis
+            array: eigenvalues
+
+        **Example**
+
+        >>> qml.Projector.compute_eigvals([0, 1])
+        [0. 1. 0. 0.]
         """
-        w = np.zeros(2 ** len(params[0]))
-        idx = int("".join(str(i) for i in params[0]), 2)
+        w = np.zeros(2 ** len(basis_state))
+        idx = int("".join(str(i) for i in basis_state), 2)
         w[idx] = 1
         return w
 
-    @classmethod
-    def _matrix(cls, *params):
-        basis_state = params[0]
-        m = np.zeros((2 ** len(basis_state), 2 ** len(basis_state)))
-        idx = int("".join(str(i) for i in basis_state), 2)
-        m[idx, idx] = 1
-        return m
```

Comment:
How come this was deleted?

Reply:
I added the method in a separate PR, and it merged it into the dev branch although the matrix refactor already introduced another one...